lk2nd/device/dts/msm8994/msm8994-huawei-angler.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "Huawei Nexus 6P";
+	qcom,msm-id = <QCOM_ID_MSM8994 0x20000>;
+	qcom,board-id = <8026 0>;
+	qcom,pmic-id = <0x10009 0x1000A 0x0 0x0>;
+};
+
+&lk2nd {
+	model = "Huawei Nexus 6P";
+	compatible = "huawei,angler";
```

Comment:
I suppose this would be correct for the one upstream?

```suggestion
	compatible = "huawei,angler";
	
	lk2nd,dtb-files = "msm8994-huawei-angler-rev-101";
```

Reply:
Yeah, should be correct to my knowledge

Reply:
FYI [`msm8994-huawei-angler-rev-101.dts` from mainline](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/arm64/boot/dts/qcom/msm8994-huawei-angler-rev-101.dts) contains:

```
compatible = "huawei,angler", "qcom,msm8994";
```

Reply:
Yes, we decided to drop the soc compatible for lk2nd since it would be pretty much the same on all nodes anyway and for us doesn't really contain much information.

So for lk2nd I ask for a slightly different convention:

- If possible, most specific compatible (i.e. for a variant of the device, even if hw is the same) which doesn't have to exist upstream.
- A chain of compatibles for this device used upstream (so i.e. either just one generic one or also more specific one if upstream has a dedicated dts for it)
- No non-device-specific compatibles (so no compatible for a SoM a device is based on (if applicable) and no soc compatible)

FWIW for now this property is mostly informational anyway but if we ever i.e. want to match a dtb based on it, "match any "qcom,msm8994" if you have nothing else" is not a great idea.. :D